mlir/test/fusion/e2e/reductions/gfx90a/rock-gemm-reduce-case1.mlir

```diff
@@ -0,0 +1,47 @@
+// This test checks a case where reduction is happening in fastest changing dimension where vectorization is present
```

Comment:
Suggestion: Use `-host-pipeline=xmodel,runner` and then `mlir-cpu-runner`. 

But also, to make this not hardcode a target name, use the arch name we have floating around the tests and do a `sed -e 's/###GPU_ARCH###/%name_of_the_variable/g` on the input file before passing it to `rocmlir-gen`. This'll also let you set `xmodel.arch`